upcloud/resource_upcloud_objectstorage_test.go

```diff
@@ -15,27 +15,24 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/minio/minio-go/v7"
+	"github.com/rs/xid"
```

Comment:
Just wondering do we need another dependency for this?  Could we just use `time.Now().Unix()` or `time.Now().UnixNano()`  to suffix variables?

Reply:
It's actually already a dependency (although an indirect one) at `go.mod`. But in case of leftovers from previous runs it would likely make things easier if you'd get some idea when the storage was created (already from the name). I'll change this to a timestamp. 👍